finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Parameter.scala

```diff
@@ -0,0 +1,37 @@
+package com.twitter.finagle.exp.mysql
+
+import com.twitter.finagle.exp.mysql.transport.BufferWriter
+import language.implicitConversions
+
```

Comment:
Some scaladoc that briefly explains what Parameter is would be nice. Note, that all usage of Parameter should be via CanBeParameter.